tensorflow_probability/python/bijectors/hypothesis_testlib.py

```diff
@@ -398,3 +402,19 @@ def frechet_constraint(loc):
   def constrain(x):
     return loc + tf.math.softplus(x)
   return constrain
+
+def gev_constraint(loc, scale, conc):
+  """
+  Maps `s` to:
+    1. [loc - scale / conc, inf) if conc > 0;
+    2. (-inf, inf) if conc = 0;
+    3. (-inf, loc - scale / conc] if conc < 0.
+  """
+  def constrain(x):
+    if conc == 0:
```

Comment:
Given that these are tensors, you'll need to use tf.where semantics here:

```suggestion
    tf.where(conc > 0., tf.math.softplus(x) + loc - scale / conc, tf.where(tf.equal(0., conc), x, loc - scale / conc - tf.math.softplus(x))
```

Reply:
Thx for your suggestion! Updated~